tests/src/system/basic/WskBasicTests.scala

```diff
@@ -168,6 +168,16 @@ class WskBasicTests
             wsk.action.delete(name, expectedExitCode = CONFLICT)
     }
 
+    it should "reject delete of an action without an action name" in {
+        wsk.cli(Seq("action", "delete"), expectedExitCode = ERROR_EXIT).
+          stderr should include("error: Invalid argument(s). An action name is required.")
```

Comment:
It gets annoying I think if we always print usage information. I suggest we should do that deliberately in a few cases only.